docs/content/howtos/simulate-ransomware.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "How to simulate a ransomware attack"
+draft: false
+pre: '<i class="fa fa-lock"></i> '
+tags: ["howtos", "simulate ransomware"]
+---
+
+Follow these steps to successfully simulate a ransomware attack on your
+network.
+
+1. Prepare your environment for a ransomware simulation
+
+    The Infection Monkey will only encrypt files that are allowed by you.
+    Prepare your network by adding a "ransomware target" directory to each
```

Comment:
Do the permissions need to be set up a certain way? Do I need to give it a certain name? Do I need to add files to the directory? Will the monkey recurse into subdirectories?